CorsixTH/Src/th_gfx_font.h

```diff
@@ -28,15 +28,18 @@ SOFTWARE.
 #include FT_FREETYPE_H
 #endif
 
+class render_target;
+
```

Comment:
empty line doesn't look useful to me

Reply:
It's consistent with a rule about leaving a blank line of white space between class definitions. Perhaps not useful in this case, but not worth an override either imho.